src/lib.rs

```diff
@@ -28,9 +28,10 @@ extern crate std;
 //pub mod attestation_types;
 
 pub mod crypto;
+pub mod error;
```

Comment:
It seems that this module is only ever used by `report` module, so why not define the `Error` enum there?